group.py

```diff
@@ -1,5 +1,9 @@
 """An example of how to represent a group of acquaintances in Python."""
 
-# Your code to go here...
-
-my_group =
+#For the relationship of each person, it is like: the front person is the “relationship” of the behind person. 
+my_group = {
+    "Jill":{"age":26,"job":"biologist","relationship":{"friend":["Zalika"],"partner":["John"]}},
+    "Zalika":{"age":28,"job":"artist","relationship":{"friend":["Jill"]}},
+    "John":{"age":27,"job":"writer","relationship":{"partner":["Jill"]}},
+    "Nash":{"age":34,"job":"chef","relationship":{"landlord":["Zalika"]}},
```

Comment:
Nash is 34, a chef, _John's cousin_ and Zalika's landlord.

Missed John here :(